plugins/aws/iam/rolePolicyUnusedServices.js

```diff
@@ -411,7 +411,8 @@ module.exports = {
                                         statement.Action[0].split(':')[1].toLowerCase();
                                     if (statement.Action.length > 1 || statement.Action[0] !== '*') {
                                         for (let action of statement.Action) {
-                                            if (config.whitelist_unused_actions_for_resources.includes(action)) continue;
+                                            if (config.whitelist_unused_actions_for_resources.includes(action) ||
+                                                (action && action.length && action === '*')) continue;
```

Comment:
There are three equaltos! Is this correct?

Reply:
Team. Please correct me if i am wrong

Reply:
Awesome! I didn;t knew about this at all. Thanks.